### PR TITLE
PP-12444 cypress test confirm org details stripe setup task

### PR DESCRIPTION
**Pull request by nlsteers**
## WHAT

- cypress test for the `confirm org details` stripe setup task

### CODE REVIEW

File: test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+          })
+
+          it('should redirect to the task summary page on success', () => {
+            cy.get('input[type="radio"]')
+              .siblings('label')
+              .contains('No, these organisation details do not match')
+              .prev('input[type="radio"]')
+              .check()
+
+            cy.get('#confirm-org-details-form button[type="submit"]').click()
```
Comment (oswaldquek) [inline test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js:223]: How is this line differentiated from [this same one above for a 'yes' selection](https://github.com/alphagov/pay-selfservice/pull/4378/files#diff-666005f7faf40ee7f1e88758b19501f4f7d86ea256202d2f5817e7b83d7ff309R169)?

File: test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            cy.location('pathname').should('contain', '/organisation-details/update')
+            cy.get('h1').should('contain', 'What is the name and address of your organisation on your government entity document?')
```
Comment (oswaldquek) [inline test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js:225]: A screenshot for this in the PR would be great.

File: test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js (hunk 1 above)
Comment (nlsteers) [reply test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js:223]: the difference being that on one test the radio 'yes' is checked before hitting the submit button and on this test the radio 'no' is checked? I'm not sure I follow.

Comment (oswaldquek) [reply test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js:223]: Ohhh sorry lol

File: test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js (hunk 2 above)
Comment (nlsteers) [reply test/cypress/integration/simplified-account/service-settings/stripe-details/confirm-org-details.cy.js:225]: screens of the task view? they were in a previous PR: https://github.com/alphagov/pay-selfservice/pull/4361

**Review by oswaldquek — APPROVED** [review]

**Pull request merged**